main/include/WeaveProjectConfig.h

```diff
@@ -38,18 +40,35 @@
  */
 #define WEAVE_DEVICE_CONFIG_ENABLE_TEST_DEVICE_IDENTITY 34
 
-// Use a default pairing code if one hasn't been provisioned in flash.
+/**
```

Comment:
I don't think this should be a doxygen comment.  This instance of the #define isn't definitive w.r.t. documenting the value.  Therefore this should probably just be a simple comment.

Reply:
fixed.